### DISCUSSION TITLE
Export only shown information

### DISCUSSION THREAD

**Discussion by xstable** [kimai/kimai#4330] [Questions & Answers] [answered]
Customer ask for the estimates of the team on each end of the month.
I go to Reporting, choose the Project and go to export.

Is it possible to configure the needed columns in any way (especially for PDF creation)?

What I currently need to do: 
1. Export the filtered data
2. open CSV and remove all columns that are not needed
3. create a pdf out of this csv-chart

Best way to solve this in my opinion would be, that only those data are exported, which are visible in the List: 

![image](https://github.com/kimai/kimai/assets/1778068/34491355-016e-486c-a24f-d6e11b138a40)

means if you need to show estimates, or hourly rates, etc. you should need to enable it in the view-settings first: 
![image](https://github.com/kimai/kimai/assets/1778068/246d5e13-5d2c-434c-8e5a-325af727abd0)

what do you think about this?

**Comment by xstable** [comment, marked as answer]
Ah, I found another way, it's only a bit hidden. 
1. Go to Projects, 
2. click on it
3. choose "Filter Data" and "Export" in the Top-menu

![image](https://github.com/kimai/kimai/assets/1778068/3559555e-ca6a-45d7-8587-6335d6938284)

You are lead to a new Screen with a Filter-Dialog similar to those in Reportings... BUT it's another one
So enter your stuff for filtering, and scroll down:

![image](https://github.com/kimai/kimai/assets/1778068/48b6a851-d526-471e-b981-736ff0c2b119)

In the Context-menu you have different options what kind of report do you want. Choose it, and you got a nice, plain listing only about the hours. 

![image](https://github.com/kimai/kimai/assets/1778068/62505e57-6d21-4834-8c8e-26aac044a120)

Looks like this: 
![image](https://github.com/kimai/kimai/assets/1778068/e38ff175-e745-4065-9b2b-eeee58e28b18)

# Resumè
It was my fault, as I didn't noticed, that the Export Section in Timetracks > Exports exactly do what I was searching for.

**Reply by kevinpapst** [reply]
Glad you could help yourself 😄
